pom.xml

```diff
@@ -93,6 +93,11 @@
 			<artifactId>logback-classic</artifactId>
 			<version>1.2.3</version>
 		</dependency>
+		<dependency>
```

Comment:
* logging, testing and caching are already provided by Spring Boot - remove those dependencies
* Spring Boot has a bunch of already specified versions, which you should not override (SLF4J, Logback)

Reply:
remove logback.xml. Anything bad with a default config?

Reply:
fixed